api/src/main/java/org/aya/api/ref/DefVar.java

```diff
@@ -54,4 +54,11 @@ private DefVar(Concrete concrete, Core core, @NotNull String name) {
   @Override public boolean equals(Object o) {
     return this == o;
   }
+
+  public boolean isInModule(@NotNull ImmutableSeq<String> moduleName) {
+    var maybeSubmodule = module;
+    if (maybeSubmodule == null) return false;
+    if (maybeSubmodule.sizeLessThan(moduleName.size())) return false;
```

Comment:
This line is probably useless

Reply:
slice-and-compare is much slower than int comprisons

Reply:
And it's not useless as `.sliceView` may throw `IndexOutOfBounds`

Reply:
I didn't realize that kala sameElements didn't compare size 🤯 @Glavo

Reply:
> I didn't realize that kala sameElements didn't compare size 🤯 @Glavo

No. it's `.sliceView`, not `.sameElements`

Reply:
Just try `SeqView.of(1).sliceView(0, 10)`